src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -133,14 +150,33 @@ private async Task UpdateImportTableRow(Guid releaseId, string dataFileName, int
             {
                 SubjectId = _guidGenerator.NewGuid(),
                 DataFileName = dataFileName,
-                OrigDataFileName = dataFileName,
                 MetaFileName = metaFileName,
                 Release = importMessageRelease,
                 NumBatches = 1,
                 BatchNo = 1,
                 ArchiveFileName = zipFileName
             };
         }
+
+        private CloudQueue GetOrCreateQueue(string queueName)
+        {
+            var client = CreateCloudQueueClient();
+            var queue = client.GetQueueReference(queueName);
+            queue.CreateIfNotExists();
+            return queue;
+        }
+
+        private CloudQueueClient CreateCloudQueueClient()
```

Comment:
We recently added Azure SDK v12 to the project. It's a completely new library, but we should probably be using this instead as it's obviously where the SDK will be moving in the future.

Reply:
Good to know.  This PR has moved on a bit since you looked at it.  I'm now "outsourcing" this work to the already-existing StorageQueueService, which I stumbled on by luck.  Looks like that's a candidate class for these changes.  If it's OK I won't try to tackle those changes here though.  New ticket?

Reply:
Yep that's fine by me :+1:

Reply:
https://dfedigital.atlassian.net/browse/EES-1740 created, thanks.